docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+Lokomotive supports Flatcar Container Linux only. [Flatcar Linux Update Operator
+(FLUO)](https://github.com/kinvolk/flatcar-linux-update-operator) can keep the operating system
+updated every time there is a new Flatcar release. FLUO is a supported Lokomotive component. This
+guide will show how to install it and disable its auto-update feature for specific nodes.
```

Comment:
I'd add more context to introduction to make it more comprehensive. 
```suggestion
Right now Lokomotive only supports [Flatcar Container Linux](https://www.flatcar-linux.org/) as underlying operating system for nodes. While Flatcar can keep itself up to date, when running Kubernetes on top of it it is recommended to use [Flatcar Linux Update Operator
(FLUO)](https://github.com/kinvolk/flatcar-linux-update-operator) to perform updates to avoid rebooting too many cluster nodes at a time, which could cause the outage for services running on your cluster.

FLUO is a supported Lokomotive component.

This guide will show how to install it and disable its auto-update feature for specific nodes, which might be useful when you run services, which require special care before shutting down (e.g. storage clusters).
```